src/shell/store/ui.ts

```diff
@@ -133,6 +145,21 @@ export const ui = createSlice({
     toggleUpdateFaviconModal(state: UIState, action: { payload: boolean }) {
       state.isUpdateFaviconModalOpen = action.payload;
     },
+    toggleContentNav(state: UIState, action: { payload: boolean }) {
```

Comment:
Yeah, it's persisted via idb since I added it on the local-storage middleware. Since the component is rendered at 2 different places and I need the states synced, I opted to put it in the redux store to avoid doing the whole JSON.parse(localStorage.get("key_name")) thing over again.